nsqadmin/http.go

```diff
@@ -732,6 +732,21 @@ func (s *httpServer) doConfig(w http.ResponseWriter, req *http.Request, ps httpr
 			if err != nil {
 				return nil, http_api.Err{400, "INVALID_VALUE"}
 			}
+		case "log_level":
+			var logLevelStr string // temporary variable in case the value is invalid
+			err := json.Unmarshal(body, &logLevelStr)
```

Comment:
To justify this opinion a bit more: "nsqlookupd_tcp_addresses" is a list so json makes sense, and "verbose" is a boolean which has non-quoted representation in json so that's convenient. But adding quotes around a single word, particularly for a manual tool like curl where you end up running `curl --data-binary '"info"' ...`, is a bit unexpected.